### DISCUSSION TITLE
"used by" section disappeared from my home page

### DISCUSSION THREAD

**Discussion by antoine-coulon** [community/community#73522] [New to GitHub] [answered]
Hello everyone,

Just wondering why my `used by` section on my [skott](https://github.com/antoine-coulon/skott) is not showing up anymore since few weeks. I didn't change anything particular in the repository configuration. Also in `Code security & analysis` > `Dependency graph` menus I don't have the dropdown to select a specific package like this is shown in the documentation.

Can anyone help me troubleshoot that please?

Thanks

**Comment by nuxy** [comment, marked as answer]
Contacted support on 11/23 regarding this issue.  Their reply is below.

> Thank you for contacting GitHub support. GitHub recently changed it so the Used by section will only show on repositories where the selected package has at least 100 dependents, as documented [here](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package).

### References
- https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package

**Reply by antoine-coulon** [reply]
@nuxy thanks a lot that explains why it disappeared, my project has nearly 30 used by only.

I felt like it was valuable even though still a small number, but fair enough now I know why :) 

Thanks for the help!

**Reply by Vanilagy** [reply]
I wonder what sparked that change. Seeing the "Used by" for my small library always cheered me up a little :)

**Reply by ItzNotABug** [reply]
Strangely enough, I can see the `Used by` (even if a very low number for it) in the embeds when sharing links on some platforms like Discord. The same isn't really visible on GitHub, sigh!

**Reply by antoine-coulon** [reply]
While I got the answer justifying why it does not show on the repo, in my opinion "used by" count should appear whatever the number is, especially if the information is accessible elsewhere, it should be consistent.

**Comment by Vanilagy** [comment]
There's actually a little "hack" I found with which you're still able to see the Used By *count* (not the specific repos that use it). Turns out this information is still sent down with the Open Graph image that embeds when you post a link to your repo in certain places. The link to this image is as follows:

```
https://opengraph.githubassets.com/5b92c81c05d64a82c3fb4ba95739403a2d38cbad61f260a0701b3366b3d10327/facebook/react
```

Replace `facebook/react` with the path to your repo to see the Used By count.

**Reply by charlieforward9** [reply]
this doesnt work anymore (at least on `leaflet` and `leaflet.glify`)

**Reply by Vanilagy** [reply]
Damn :/

**Comment by hassannasir1236** [comment]
It seems like the "Used by" section and Dependency Graph dropdown are not showing up in your repository. Here are some troubleshooting steps:

Check Repository Visibility: Ensure the repository is public, as features like "Used by" and the Dependency Graph are available only for public repos.
Verify Dependency Graph: Ensure your project includes supported files (like package.json or requirements.txt) and has proper dependency metadata.
Re-enable Features: Go to Settings > Code security & analysis > Dependency graph, and toggle it off and on again.
Check GitHub Status: There could be service disruptions—check [GitHub Status](https://www.githubstatus.com/) for updates.
If the issue persists, reach out to GitHub support.

**Reply by Vanilagy** [reply]
🤖

**Reply by samus-aran** [reply]
Hi @hassannasir1236

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
